addon/mixins/flexberry-map-load-layers.js

```diff
@@ -0,0 +1,87 @@
+/**
+  @module ember-flexberry-gis
+ */
+
+import Ember from 'ember';
+import QueryBuilder from 'ember-flexberry-data/query/builder';
+import FilterOperator from 'ember-flexberry-data/query/filter-operator';
+
+/**
+  Mixin for recursive load map layers from store
+  @class FlexberryMapLoadLayersMixin
+  @uses <a href="http://emberjs.com/api/classes/Ember.Mixin.html">Ember.Mixin</a>
+ */
+export default Ember.Mixin.create({
+
+  /**
+    @return {Ember.RSVP.Promise} Promise with loading flexberry-map-layer
+    @private
+   */
+  _loadLayer(layer) {
```

Comment:
All private methods in this mixin are contextless, they don't use any methods or properties from current instance (this).
So there is no need for them to be a mixin members. They can be placed outside of the mixin.